src/Axiom/Set.agda

```diff
@@ -167,13 +167,13 @@ record Theory {ℓ} : Type (sucˡ ℓ) where
   ∈-unions : {a : A} {U : Set (Set A)} → (∃[ T ] T ∈ U × a ∈ T) ⇔ a ∈ proj₁ (unions U)
   ∈-unions = proj₂ $ unions _
 
-  ∅ : Set A
-  ∅ = fromList []
+  ∅ˢ : Set A
+  ∅ˢ = fromList []
 
-  ∅-strongly-finite : strongly-finite {A} ∅
+  ∅-strongly-finite : strongly-finite {A} ∅ˢ
```

Comment:
The goal of this PR is to rename `∅ ` to `∅`.  To that end, I had to change `∅` to `∅ˢ` in this file (and in a few other places) to accommodate the new type classes `HasEmptySet` and `HasEmptyMap`.  Fortunately, I am able to limit the superscripts to blocks of code in `.agda` files or *hidden* blocks in `.lagda` files.